frontend/src/metabase/sharing/components/SharingSidebar.jsx

```diff
@@ -304,7 +290,9 @@ class SharingSidebar extends React.Component {
 
   editPulse = (pulse, channelType) => {
     this.setPulse(pulse);
-    this.setState({ editingMode: "add-edit-" + channelType });
+    this.setState({
+      editingMode: "add-edit-" + channelType,
```

Comment:
Concatenating strings for a enum-esque usage makes me nervous, but it's probably fine.

Reply:
Yeah, I'm in general not a fan of how `editingMode` works but am fixing the even more broken stuff first :stuck_out_tongue: